nucleus/admin/cli/src/main/java/com/sun/enterprise/admin/cli/CLICommand.java

```diff
@@ -1440,12 +1440,25 @@ protected void buildTerminal() {
     
     protected void buildLineReader() {
         if (lineReader == null) {
-            lineReader = LineReaderBuilder.builder()
+            lineReader = newLineReaderBuilder()
                     .terminal(terminal)
                     .build();
         }
     }
     
+    protected LineReaderBuilder newLineReaderBuilder() {
+        // In community this should be disabled by default
+        boolean disabled = true;
+        Environment environment = new Environment();
```

Comment:
The command already contains the env field as @Pandrex247 pointed out. We should use it instead of creating a new instance:

```suggestion
        Environment environment = this.env;
```